app/assets/javascripts/modules/project.js

```diff
@@ -30,4 +30,45 @@ angular.module('probedock.project', [ 'probedock.api', 'probedock.forms', 'probe
     });
   })
 
+  .directive('projectRecentActivity', function() {
+    return {
+      restrict: 'E',
+      controller: 'ProjectRecentActivityCtrl',
+      controllerAs: 'ctrl',
+      templateUrl: '/templates/project-recent-activity.html',
+      scope: {
+        organization: '=',
+        project: '='
+      }
+    };
+  })
+
+  .controller('ProjectRecentActivityCtrl', function(api, $scope) {
+
+    $scope.$watch('project', function(value) {
+      if (value) {
+        fetchReports();
+      }
+    });
+
+    function fetchReports() {
+      return api({
+        url: '/reports',
+        params: {
+          pageSize: 5,
+          organizationId: $scope.organization.id,
```

Comment:
The `organizationId` here is superfluous (but indeed required with the current version of the resource). By copying the following improvement to the reports API resource, you can remove it: https://github.com/probedock/probedock/blob/77bcad1e5ca75b86361412bd113e945a76646db4/app/api/probe_dock/metrics_api.rb#L31-L32